### PR TITLE
Feature re form

### PR DESCRIPTION
**Pull request by atenaiis**
Team,

In this project the aim was to work with forms usig rails.
Please follow the Readme file instructions.
I worked on git flow.

cheers.
Mariana A. Campos

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(:username => params[:username], :email => params[:email], :password => params[:password])
```
Comment (MahmoudBakr23) [inline app/controllers/users_controller.rb]: Delete line 7, since you don't need it!

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
# **Changes Required 🛑♻**

<hr>

Hello there :)
You've done a great job so far working on this project.
Controllers, models, Linter, README, commits, and PR are well described and configured. Well done! However, there are still a few more changes to do before moving on.

- [x] Delete commented-out code from the users' controller. See my comment on your code.
- [x] You should leave the first two forms that you've created while building the project commented-out! Please attach the first form action and the second form tag commented-out in your new file.
- [x] Please add in your README file the CLI ```rails db:create``` before the ```rails db:migrate```
- **OPTIONAL** You may want to set the root page of your app's routes to 'users#new' instead of the original Ruby On Rails starter page.
<hr>

Kindly do the required changes, then submit for another CRR.
Please feel free to contact me on Slack with the name "Mahmoud Bakr" if you have any question 👍 
Good luck and happy coding 👨‍💻💻

**Comment by atenaiis** [comment]
Thanks for that quick review,

I've implemented the changes required.

Cheers.

Mariana A. Campos

**Review by smcommits — APPROVED** [review]
# Status: Approved :+1: 
## Hello, @atenaiis! You have done a great job. The application appears to be fantastic. Good work implementing the changes requested by the previous TSE. This PR is now approved. You can go ahead and merge the branch. Congratulations. 

## Optional 
**Please note that the following suggestions are optional**

- [x] Try to write more descriptive commit messages. 
- [x] You can create a partial for user form and reuse it in edit and new view. You can pass in the arguments for any dynamic value.

**Pull request merged**